src/assets/js/main.js

```diff
@@ -1,18 +1,21 @@
 /*
  * Polyfills
  */
-import './polyfills/object-fit';
+import "./polyfills/object-fit";
 
 /*
  * Modules
  */
 
-import './modules/handle-mobile-nav';
```

Comment:
minting hell. different linter configs? does not look like this repo even has any listing stuff defined. should 100% fix the with the CMS version of the website!